conf/apiv1.routes

```diff
@@ -1,10 +1,13 @@
 # Images list
-GET     /images          controllers.apiv1.APIv1Controller.images(query: Option[String] ?= None, offset: Int ?= 0, limit: Int ?= 25)
+GET     /images               controllers.apiv1.APIv1Controller.images(query: Option[String] ?= None, offset: Int ?= 0, limit: Int ?= 25)
 
 # A single image
-GET     /image/:id       controllers.apiv1.APIv1Controller.image(id: String)
+GET     /image/:id            controllers.apiv1.APIv1Controller.image(id: String)
+
+# Similiar images
```

Comment:
All the code is gud but it's not `similiar` in English, but `similar`! (see [wiktionary](https://en.wiktionary.org/wiki/similar#English))

Reply:
Oh. Yeah. In all other places it is correct, but not here.

Reply:
Apparently not, just a Ctrl+F on this PR's changes shows 18 matches for `similiar`!

Reply:
Oh. Wait. Not it's wrong everywhere..